components/ui/Toggle/Toggle.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Toggle } from "./Toggle";
+
+export default {
+  title: "UI/Toggle",
+  component: Toggle,
+} as ComponentMeta<typeof Toggle>;
+
+const Template: ComponentStory<typeof Toggle> = (args) => <Toggle {...args} />;
+
+export const ToggleStory = Template.bind({});
```

Comment:
when I click the toggle in the [preview env](https://storybook-722thiizm-ninjha01.vercel.app/?path=/story/ui-toggle--toggle-story&globals=darkMode:false) it doesn't toggle. Could we get that functionality in?